core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -69,6 +73,38 @@
       " (" + CATALOG_NAME + ", " + TABLE_NAMESPACE + ", " + TABLE_NAME +
       ", " + METADATA_LOCATION + ", " + PREVIOUS_METADATA_LOCATION + ") " +
       " VALUES (?,?,?,?,null)";
+
+  // Catalog Namespace
+  protected static final String CATALOG_NAMESPACE_TABLE_NAME = "iceberg_namespaces";
+  protected static final String NAMESPACE_NAME = "namespace_name";
+  protected static final String NAMESPACE_METADATA = "metadata";
+  protected static final String NAMESPACE_PROPERTIES_TABLE_NAME = "namespace_properties";
+  protected static final String NAMESPACE_PROPERTY_KEY = "key";
+  protected static final String NAMESPACE_PROPERTY_VALUE = "value";
+
+  protected static final String CREATE_NAMESPACE_TABLE =
+      "CREATE TABLE " + CATALOG_NAMESPACE_TABLE_NAME +
+          "(" +
+          CATALOG_NAME + " VARCHAR(255) NOT NULL," +
+          NAMESPACE_NAME + " VARCHAR(255) NOT NULL," +
+          NAMESPACE_METADATA + " VARCHAR(65535)," +
```

Comment:
Sometimes, an argument is used with a different name because it shadows a field in the class, which would break checkstyle.

So if the class has a field `properties`, then an argument calld `properties` would make references to `properties` ambiguous. So our checkstyle doesn't allow it (outside of constructors where it's not ambiguous and `this.` has to be used to reference the class's value).

This is often why they're different. Also, in some cases it's just whomever wrote the function signature and they were different over time because of authors or just word choice of the person when they were working on that code🤷 . But in my experience, it's usually that the arguments name would shadow a field on the class.